assets/src/story-embed-block/block/embedControls.js

```diff
@@ -39,6 +39,7 @@ const POSTER_ALLOWED_MEDIA_TYPES = ['image'];
 const FallbackComponent = ({ children }) => <>{children}</>;
 
 const { Button, BaseControl, TextControl, PanelBody, PanelRow } = Components;
+// ToolbarGroup is only available in Gutenberg 7.0 or later, so it does not exsit in WP 5.3.
```

Comment:
Whoops I made a typo, apologies!

```suggestion
// ToolbarGroup is only available in Gutenberg 7.0 or later, so it does not exist in WP 5.3.
```